packages/desktop/package.json

```diff
@@ -80,7 +80,7 @@
   },
   "homepage": "https://github.com/TryQuiet",
   "@comment version": "To build new version for specific platform, just replace platform in version tag to one of following linux, mac, windows",
-  "version": "1.2.0",
+  "version": "1.2.0-alpha.31",
```

Comment:
Isn't this version behind the full 1.2.0?

Reply:
right